test/handle_test.cpp

```diff
@@ -242,9 +242,10 @@ void test_arch_name()
                        "gfx900",
                        "gfx803",
                        "gfx1030",
+                       "gfx1031",
                        "gfx1100",
-                       "gfx1102",
-                       "gfx1031"};
```

Comment:
Please roll back all gfx1031 related changes. Unofficial support for gfx1031 (Navi 22, Radeon RX6700) has been introduced in #1531

Reply:
@atamazov we are not removing it, just reordered the list :)

Reply:
@junliume Oops 🤕